alpine-minimal: openblas fails: linux/unistd.h: No such file or directory

```
  [openblas-0.3.28]   [spkg-install] memory.c:157:10: fatal error: linux/unistd.h: No such file or directory
  [openblas-0.3.28]   [spkg-install]   157 | #include <linux/unistd.h>
  [openblas-0.3.28]   [spkg-install]       |          ^~~~~~~~~~~~~~~~
```
https://github.com/passagemath/passagemath/actions/runs/14942372970/job/41981313044#step:11:6326

--- a/build/pkgs/_prereq/distros/alpine.txt
+++ b/build/pkgs/_prereq/distros/alpine.txt
@@ -20,3 +20,5 @@ ca-certificates
 coreutils
 flex
 bison
+# needed by openblas
+linux-headers

